app.rb

```diff
@@ -4,11 +4,13 @@
 require_relative 'rental'
 require_relative 'classroom'
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
- [ ] Great job working on the requirements for preserving the data but as I can see here you disable the Linters but the Linters are very important because they Help Check for Errors, Linters can identify certain programming errors earlier on that you may not have noticed while writing code and they make your code KISS ad DRY and YAGNI so kindly solve the issues instead of disabling them 🔥🙏🏻⭐🚀✅

Reply:
Hey. I cannot write entire class within 150 lines. So, I have disabled it. 
Kindly reconsider.